---
redirect_from:
  - "/02abstract/abstract"
title: |-
  Abstract
pagenum: 1
prev_page:
  url: /01Introduction/intro.html
next_page:
  url: /03Code/Figure2.html
suffix: .md
search: sup colorado boulder br overlap daniel larremore populations objects population uncertainty estimates b university united states america between sampling species quantifying method problem department computer science biofrontiers institute imeasuring principle straightforward upon fully both shared taxonomical units gene variants depending context directly counted practice however only fraction likely sampled due stochastic data collection sequencing techniques although methods exist under subsampled conditions bias well documented cannot quantified here derive validate rigorously estimate incomplete samples total genes known special case general diversity particularly relevant ecology genomic epidemiology malaria solving bayesian inference takes into account rates subsampling produces unbiased bayes optimal addition provides

comment: "***PROGRAMMATICALLY GENERATED, DO NOT EDIT. SEE ORIGINAL FILES IN /content***"
---

    <main class="jupyter-page">
    <div id="page-info"><div id="page-title">Abstract</div>
</div>
    <div class="jb_cell">

<div class="cell border-box-sizing text_cell rendered"><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<hr>
<p><em>Daniel B. Larremore</em> <sup>1,2</sup></p>
<p><sup>1</sup> Department of Computer Science, University of Colorado Boulder, Boulder, Colorado, United States of
America
<br />
<sup>2</sup> BioFrontiers Institute, University of Colorado Boulder, Boulder, Colorado, United States of
America
<br /></p>
<p><i>"Measuring the overlap between two populations is, in principle, straightforward. Upon fully sampling both populations, 
the number of shared objects — species, taxonomical units, or gene variants, depending on the context — can be directly counted. 
In practice, however, only a fraction of each population’s objects are likely to be sampled due to stochastic data collection or 
sequencing techniques. Although methods exist for quantifying population overlap under subsampled conditions, their bias is well 
documented and the uncertainty of their estimates cannot be quantified. Here we derive and validate a method to rigorously estimate 
the population overlap from incomplete samples when the total number of objects, species, or genes in each population is known, a 
special case of the more general β-diversity problem that is particularly relevant in the ecology and genomic epidemiology of malaria. 
By solving a Bayesian inference problem, this method takes into account the rates of subsampling and produces unbiased and Bayes-optimal 
estimates of overlap. In addition, it provides a natural framework for computing the uncertainty of its estimates, and can be used 
prospectively in study planning by quantifying the tradeoff between sampling effort and uncertainty."</i></p>
<p><br />
<br /></p>
<p><strong>Corespondence</strong>: Daniel B. Larremore
Email: daniel.larremore@colorado.edu</p>

</div>
</div>
</div>
</div>

 


    </main>
    